engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-add-subscriptions.controller.js

```diff
@@ -26,7 +26,12 @@ angular.module('Bastion.activation-keys').controller('ActivationKeyAddSubscripti
             'organization_id': CurrentOrganization,
             'search': $location.search().search || "",
             'sort_order': 'ASC',
-            'available_for': 'activation_key'
+            'available_for': 'activation_key',
+            'resourceName': 'Subscriptions',
+            'emptyRowsMessage': "You currently don't have any Subscriptions to subscribe to," +
+                                " you can add Subscriptions by selecting 'Products' under" +
+                                " 'Content' in the main menu"
```

Comment:
See https://github.com/Katello/bastion/pull/116#discussion_r70081611 for my suggestion on how we should do this.